tests/test_configurations/test_base.py

```diff
@@ -1123,3 +1131,138 @@ def test_component_id_from_json():
         "version": "1.0.0",
     }
     assert ComponentId.from_json(json_data).json == json_data
+
+
+class TestConfigurationContainingPathSerialization:
+    """Test configurations containing paths are deterministic across different OS"""
+
+    def setup(self):
+        """Setup test"""
+
+        # starting from posix-compliant format
+        self.raw_paths = [
+            "C:/Documents/Newsletters/Summer2018.pdf",
```

Comment:
Re your comment on removing the root, afaic we use paths which are relative to the project so I don't think that should be an issue

Reply:
that would make sense if true but I'm not so sure. For example, we also store paths to private key files. In that is what we want to opt for though, I suggest we enforce it. We can change ticket #440 to address this, by introducing a check that ensures all paths are relative to specific root (e.g. the project)